contract/vm.go

```diff
@@ -839,8 +850,11 @@ func Call(
 	ce := newExecutor(contract, contractAddress, ctx, &ci, ctx.curContract.amount, false, false, contractState)
 	defer ce.close()
 
+	startTime := time.Now()
 	// execute the contract call
 	ce.call(callMaxInstLimit, nil)
+	vmExecTime := time.Now().Sub(startTime).Microseconds()
+	vmLogger.Trace().Int64("execµs", vmExecTime).Stringer("txHash", types.LogBase58(ce.ctx.txHash)).Msg("tx execute time in vm")
```

Comment:
is this still needed? can it be enabled only on debug mode? or maybe just remove it

Reply:
is it useful to keep this time measurement?
maybe it can be commented to avoid consuming resources